### PR TITLE
Prepare 0.3.0 Release

### PR DESCRIPTION
**Pull request by manoelmarques**

### CODE REVIEW

**Review by mrossinek — CHANGES_REQUESTED** [review]
We need to await the merging of #365 and #430 before releasing 0.3.0

**Comment by manoelmarques** [comment]
Yes, I created those PRs in all 4 repos to be prepared when Terra releases but they will be merged only when the repos are ready.

File: docs/conf.py [hunk 1]
```diff
@@ -110,7 +110,7 @@
 templates_path = ["_templates"]
 html_css_files = ["style.css", "custom.css", "gallery.css"]
 
-nbsphinx_timeout = 2400
+nbsphinx_timeout = 360
```
Comment (mrossinek) [inline docs/conf.py:113]: Can we change this back?

Comment (manoelmarques) [reply docs/conf.py:113]: Once the performance bug was fixed in Terra I put back the old values in all repositories. This helps us identify another bug like that one.

**Review by mrossinek — APPROVED** [review]

**Pull request merged**